### DISCUSSION TITLE
Is it possible to default the ListView to a select few applications?

### DISCUSSION THREAD

**Discussion by tiagomneves-pt** [davatorium/rofi#1938] [Q&A]
I have this question by some time now and I've tried different solutions to no avail. I would like to always have the same applications on _drun_ before searching for something. 

Does anyone have something familiar on config?

**Comment by DaveDavenport** [comment]
Not exactly sure about the question, given you cannot select multiple applications.
But maybe adding a default search query?

```bash
rofi -show drun -filter "Firefox ESR"
```
